next/components/forms/segments/AccountNavBar/AccountNavBar.tsx

```diff
@@ -56,23 +85,23 @@ export const AccountNavBar = ({ className, currentLanguage, sectionsList }: IPro
         </div>
         {isLogin && sectionsList && (
           <div className="border-t border-gray-200 max-w-screen-1.5lg m-auto h-[57px] w-full items-center justify-between lg:flex">
-            <ul className="w-full h-full flex items-center">
+            <div className="w-full h-full flex items-center">
```

Comment:
this should be an unordered list, with each link wrapped in list item

in fact, you should follow all of these - https://www.w3.org/WAI/tutorials/menus/structure/#menu-representation